lib/aurora_bootstrap_parallelization/runner.rb

```diff
@@ -2,6 +2,12 @@ module AuroraBootstrapParallelization
   class Runner
     def initialize( opts )
       @config = YAML.load_file( opts[ :config ] )[ "exporters" ]
+      @is_cron_job = false
+      if opts[ :cronjob ].nil?
+        @is_cron_job = false
+      else
+        @is_cron_job = opts[ :cronjob ]
+      end
```

Comment:
can this be simplified to something like 
```ruby
  @is_cron_job = opts[ :cronjob ]
```
`nil` is considered falsey in Ruby, so you can safely check against it the same you would for `false`. Or is there another reason for this check?

Reply:
Sure